### PR TITLE
Bump package:http to v1 in packages

### PR DESCRIPTION
**Pull request by shilangyu**
Since we are requiring dart 3 here, maybe it is worth revisiting the API to modernize for records/class modifiers?

### CODE REVIEW

File: .github/workflows/cqrs-publish.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Setup Dart
         uses: dart-lang/setup-dart@v1
         with:
-          sdk: 2.18.6
+          sdk: 3.0.1
```
Comment (bartekpacia) [inline .github/workflows/cqrs-publish.yml:27]: Here you don't use quotes, but in other places you do.
Is there any difference?

File: packages/cqrs/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# 9.0.0
+
+- Bumped `http` dependency to `1.0.0`.
+- **Breaking:** Bump minimum Dart version to 3.0.
```
Comment (bartekpacia) [inline packages/cqrs/CHANGELOG.md:4]: I like linking PRs in changelogs, like this:

```
- Bumped `http` dependency to `1.0.0` (#105)
- **Breaking:** Bump minimum Dart version to 3.0 (#105)
```

But that's just my quirk, I guess.

File: packages/cqrs/pubspec.yaml [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
+version: 9.0.0
 homepage: https://github.com/leancodepl/flutter_corelibrary/tree/master/packages/cqrs
 repository: https://github.com/leancodepl/flutter_corelibrary
 description: >-
   A library for convenient communication with CQRS-compatible backends, using
   queries and commands.
 
 environment:
-  sdk: ">=2.17.0 <3.0.0"
+  sdk: '>=3.0.1 <4.0.0'
```
Comment (bartekpacia) [inline packages/cqrs/pubspec.yaml:10]: why 3.0.1 instead of 3.0.0?

File: packages/login_client/pubspec.yaml [hunk 4]
```diff
@@ -8,17 +8,17 @@ description: >-
   refreshes tokens.
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: '>=3.0.1 <4.0.0'
```
Comment (bartekpacia) [inline packages/login_client/pubspec.yaml:11]: ditto

**Review by bartekpacia — COMMENTED** [review]
some comments

File: packages/cqrs/pubspec.yaml (hunk 3 above)
Comment (shilangyu) [reply packages/cqrs/pubspec.yaml:10]: Formatting difference

File: packages/login_client/pubspec.yaml (hunk 4 above)
Comment (shilangyu) [reply packages/login_client/pubspec.yaml:11]: As above

File: .github/workflows/cqrs-publish.yml (hunk 1 above)
Comment (shilangyu) [reply .github/workflows/cqrs-publish.yml:27]: I am no yaml expert, but I think both produce strings? I just replaced whatever was there already. We can change every occurrence to be consistent

File: packages/cqrs/CHANGELOG.md (hunk 2 above)
Comment (shilangyu) [reply packages/cqrs/CHANGELOG.md:4]: It is definitely a good idea, easier to track down the change

File: .github/workflows/cqrs-publish.yml (hunk 1 above)
Comment (bartekpacia) [reply .github/workflows/cqrs-publish.yml:27]: I prefer to not use quotes in YAML wherever possible, but when it comes to semver versions such as this, I like to put single quotes around it. I recall there are some wild problems with sth like `0.2.3` being treated as octal (I honestly don't remember).

Comment (shilangyu) [reply .github/workflows/cqrs-publish.yml:27]: Would you like me to change all occurrences of version strings to be quoted?

**Comment by Albert221** [comment]
`coverage` 1.3.0 change:
> Add a --package flag, which takes the package's root directory, instead of the .package file. Deprecate the --packages flag.

File: .github/workflows/cqrs-publish.yml (hunk 1 above)
Comment (shilangyu) [reply .github/workflows/cqrs-publish.yml:27]: friendly ping @bartekpacia

Comment (bartekpacia) [reply .github/workflows/cqrs-publish.yml:27]: Sorry for the lag.

I'd prefer to not use quotes at all, and only add them if something breaks.

**Review by bartekpacia — APPROVED** [review]

**Pull request merged**